experimental/report/testdata/i18n.yaml.color.txt

```diff
@@ -1,8 +1,8 @@
 ⟨b.red⟩error: emoji, CJK, bidi⟨reset⟩
 ⟨blu⟩  --> foo.proto:5:9
    | 
-⟨blu⟩ 5 | ⟨reset⟩message 🐈‍⬛ {
-⟨blu⟩   | ⟨reset⟩⟨reset⟩        ⟨b.red⟩^^⟨reset⟩ ⟨b.red⟩
+⟨blu⟩ 5 | ⟨reset⟩message 🐈<U+200D>⬛ {
```

Comment:
Because terminals. I did some research and found that some terminals do not correctly render all grapheme clusters, including many emoji presentation sequences. `🐈‍⬛` was being rendered as `🐈⬛` which causes all sorts of issues when highlighting spans. I decided to follow what Rust does and explicitly print all unprintable characters. Most grapheme clusters (e.g., those that use combining marks) are unaffected.